sps20/content/contents.lr

```diff
@@ -37,6 +37,9 @@ Take a look at our [recordings](/recordings) and explore what was covered in pre
 Do you want to support Swiss Python Summit? Find out more about [sponsoring opportunities](./sponsoring#sponsoring-2025).
 
 #### Gold
+<div style="display: flex; align-items: center">
+    <a href="https://amanox.ch/"><img src="/sponsors/axians_amanox.png" style="width: 120px;"></a>
+</div>

```

Comment:
Since they are the gold sponsor, I'd display their logo larger:

old
![image](https://github.com/user-attachments/assets/44d0e0e2-e08d-4393-8915-748f6f1d9461)

new
![image](https://github.com/user-attachments/assets/0dabe3b5-c50c-4e90-9ee1-8a76ae5dd013)


```suggestion
<div style="display: flex; align-items: center">
    <a href="https://amanox.ch/"><img src="/sponsors/axians_amanox.png" style="width: 240px;"></a>
</div>

```